### DISCUSSION TITLE
isA query issue on openresearch

### DISCUSSION THREAD

**Discussion by MusaabKh** [SmartDataAnalytics/OpenResearch#98] [Q&A] [answered]
Hey, the complete copy of openresearch yeilds 18000 pages. Is that normal? I used the modification date query as isA wasnt working on openreasearch. 
The query used which yeilded 0 results:
wikipush -s or -t ormk -q "[[isA::Event series]]" 
The query used to create complete copy:
wikipush -s or -t ormk -q "[[Modification date::+]]" -qd 1000 --progress

**Comment by WolfgangFahl** [comment, marked as answer]
We just recently changed the Sidebar of open research to reflect the decision to focus on Event and EventSeries see
https://www.openresearch.org/wiki/Development_guidelines and https://www.openresearch.org/mediawiki/index.php?title=MediaWiki%3ASidebar&type=revision&diff=228545&oldid=228168

The following 3 wikipush commands would give you the gist of OPENRESEARCH pages. We'll consider all other pages as corner cases for the time being. An Exception are pages which are necessary to keep things working. Which pages are necessary is part of the idea of working with the 8  OPENRESEARCH test and development copies we currently have at:

- https://or.bitplan.com/index.php/Main_Page
- https://confident.dbis.rwth-aachen.de/wikicms/
- https://www.confident-conference.org/r/wikicms/

```
wikipush -l -s or -t test -q "[[isA::Event]]"
copying 8848 pages from or to test
1/8848 (   0%): copying 3DUI 2020 ...✅
2/8848 (   0%): copying 3IA 2009 ...✅
3/8848 (   0%): copying 3PGIC 2010 ...✅
4/8848 (   0%): copying 4S4D 2017 ...✅
5/8848 (   0%): copying 5GU 2017 ...✅
6/8848 (   0%): copying 5GWN 2017 ...✅
7/8848 (   0%): copying AAAI 1980 ...✅
8/8848 (   0%): copying AAAI 1982 ...✅
9/8848 (   0%): copying AAAI 1983 ...✅
10/8848 (   0%): copying AAAI 1984 ...✅
11/8848 (   0%): copying AAAI 1986 ...✅
...
wikipush -s or -t test -q "[[Category:Event series]]"
copying 1029 pages from or to test
1/1029 (   0%): copying 3DUI ...✅
2/1029 (   0%): copying AAAAECC ...✅
3/1029 (   0%): copying AAAI ...✅
4/1029 (   0%): copying AAMAS ...✅
5/1029 (   0%): copying ACAH ...✅
6/1029 (   1%): copying ACC ...✅
7/1029 (   1%): copying ACCV ...✅
8/1029 (   1%): copying ACE (DIMEA) ...✅
9/1029 (   1%): copying ACIIDS ...✅
10/1029 (   1%): copying ACIVS ...✅
11/1029 (   1%): copying ACL ...✅
wikipush -s or -t test -p "Property:isA" "Property:used for" "Property:Acronym" "Property:Title" "Property:Start date" "Property:End date" "Property:Homepage" -f
copying 5 pages from or to test
1/5 (  20%): copying Property:Acronym ...✅
2/5 (  40%): copying Property:Title ...✅
3/5 (  60%): copying Property:Start date ...✅
4/5 (  80%): copying Property:End date ...✅
5/5 ( 100%): copying Property:Homepage ...✅
```

**Reply by WolfgangFahl** [reply]
see https://github.com/SmartDataAnalytics/OpenResearch/issues/95 for the deprecation note of some of the entities e.g. Person and Organization - we also deprecated Tools and others ...

**Reply by WolfgangFahl** [reply]
You also seem to have hit the reason for #101 that is your query might not have worked because of the blank in "Event series".
